projects/cdk/services/breakpoint.service.ts

```diff
@@ -0,0 +1,78 @@
+import {ChangeDetectorRef, Inject, Injectable, InjectionToken, Self} from '@angular/core';
+import {tuiWatch} from '@taiga-ui/cdk/observables';
+import {Observable} from 'rxjs';
+import {distinctUntilChanged, startWith, takeUntil} from 'rxjs/operators';
+
+import {TuiDestroyService} from './destroy.service';
+
+// will replace after resolving dependency cycle issue
+const TUI_MEDIA = new InjectionToken<TuiMedia>(`[TUI_MEDIA]: Token for media constant`, {
```

Comment:
@waterplea  @splincode 
I am not able to:
import {TUI_MEDIA} from '@taiga-ui/core';

It results in dependency cycle. How to resolve this ? 🤔

Reply:
`CDK` package should not depend on something from `CORE`.

I think that you should move your new service to `CORE` package.